### PR TITLE
Waste Container Schema. Refining id character restriction

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

**Comment by jmcanterafonseca** [comment]
@gtorodelvalle r?

@fgalan can you have a look at the reg exp for ids?

thanks

File: common-schema.json [hunk 1]
```diff
@@ -3,12 +3,17 @@
   "id": "https://fiware.github.io/dataModels/common-schema.json",
   "title": "Common definitions for FIWARE Harmonized Data Models",
   "definitions": {
+    "EntityIdentiferType": {
+      "type": "string",
+      "minLenght": 1,
```
Comment (kzangeli) [inline common-schema.json]: Typo: minLenght    ... th

File: common-schema.json [hunk 2]
```diff
@@ -3,12 +3,17 @@
...
+      "maxLength": 256,
+      "pattern": "^[\\w\\-:\\.,;\\(\\)/\\{\\}]+$"
```
Comment (fgalan) [inline common-schema.json]: Two comments:

* Check that "all symbols in the ACCII basic set" minus `[&, ?, /. #]` is equal to the symols added to `\w` in this line
* This regex is aligned with the rules at NGSIv2 spec for ID? Or with the rules at NGSIv2 spec for ID + specific Orion implementation rules (that adds additional forbidden chars to the NGSIv2 spec basic set)?

Comment (fgalan) [inline common-schema.json]: @jmcanterafonseca offline confirms that is the second case, so the regex should also forbid the following chars:


*    <
*    >
*    "
*    '
*    =
*    ;
    (
    )

File: common-schema.json (hunk 1 above)
Comment (jmcanterafonseca) [inline common-schema.json]: well spotted, unfortunately JSON Schema validators didn't warn about that

Comment (fgalan) [inline common-schema.json]: Isn't there any JSON Schema to validate the JSON Schema itself :)?

Comment (jmcanterafonseca) [inline common-schema.json]: fixed https://github.com/Fiware/dataModels/pull/100/commits/a6b3f5b43a2d3ffb11779abd01a8f58eca951324

File: common-schema.json (hunk 2 above)
Comment (jmcanterafonseca) [inline common-schema.json]: fixed https://github.com/Fiware/dataModels/pull/100/commits/a6b3f5b43a2d3ffb11779abd01a8f58eca951324

**Comment by gtorodelvalle** [comment]
Merging after 2 LGTMs... :)

**Pull request merged**